README.md

```diff
@@ -2,28 +2,65 @@
 
 A JavaScript library for managing authentication for [Storyblok](https://www.storyblok.com) apps.
 
+## Migrating v1 to v2
+
+The `@storyblok/app-extension-auth` v1 stored the access token in a cookie. However, v2 now offers an adapter pattern, allowing you to store the access token elsewhere, such as in a dedicated database. And here are the breaking changes:
```

Comment:
I am wondering if this is a bit misleading, as we abstracted the logic to the adapter pattern however the storage inside a DB and enabling passing a custom adapter is not possible at the moment right?

Reply:
I'm not sure if I follow you. How is it not possible?

Reply:
Currently the user cannot pass their own adapter to authHandler or the sessionStore.

Reply:
Oh I didn't realize we removed adapter from AuthHandlerParams. I'll modify this.